adjust cache path to new path as of genshin version 4.0

TheProJeckt opened the pull request:
I have updated the path to the cache folder for the 4.0 version of the Game, so Wishing Well works again.
And I bumped the Version to "1.4.1".

I haven´t created a release file yet.

TheProJeckt committed:
adjust cache path to new path as of genshin version 4.0

TheProJeckt commented:
Added a release file in my repo. For all of the people who can’t wait. ^^

https://github.com/TheProJeckt/wishing-well/releases/tag/v1.4.1

FichteFoll commented on wishing_well/util.py:
```diff
@@ -75,7 +75,7 @@ def get_cache_path():
     # python cannot do this without raising an error, and neither can the default
     # windows copy command, so we instead delegate this task to powershell's Copy-Item
     try:
-        path = Path(game_path) / 'GenshinImpact_Data/webCaches/2.13.0.1/Cache/Cache_Data/data_2'
```
Since it is likely that this is going to change in the future, it would be better if the code automatically picked the correct folder instead of requiring code changes after every patch.

FichteFoll replied:
I'll submit a fix with this code on the weekend as I already wrote one for warp-journal.

TheProJeckt committed:
adjust cache path to new path as of genshin version 4.1

I have updated the path to the cache folder for the 4.1 version of the Game, so Wishing Well works again.

TheProJeckt committed:
bumped the Version to "1.4.2"

I have updated the path to the cache folder for the 4.1 version of the Game, so Wishing Well works again.
And I bumped the Version to "1.4.1".
